cmd/root.go

```diff
@@ -66,6 +66,7 @@ func NewCommand() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&config.Output.File, "output-file", "", "File in module directory to insert output into (default \"\")")
 	cmd.PersistentFlags().StringVar(&config.Output.Mode, "output-mode", "inject", "Output to file method ["+cli.OutputModes+"]")
 	cmd.PersistentFlags().StringVar(&config.Output.Template, "output-template", cli.OutputTemplate, "Output template")
+	cmd.PersistentFlags().BoolVar(&config.Output.Check, "output-check", false, "check if output is generated doc (default false)")
```

Comment:
I think this is bit vague. Maybe something like:

```text
Check if content of output file is up to date
```

Reply:
Sounds good to me